### ISSUE TITLE
Any support on cities worldwide?

### ISSUE THREAD

**Issue by playground** [sjlu/cities#12]
Will it be available soon?

**Comment by sjlu** [comment]
No. Not at the moment. I don't have a reliable source to pull in data to do
it worldwide. If you have a specific country in mind I may consider doing a
one-off.

**Comment by playground** [comment]
Hi @sjlu,
Can the following source be used?

I'm looking https://www.npmjs.com/package/offline-geocoder & https://github.com/OpenBookPrices/country-data

**Comment by sjlu** [comment]
http://www.geonames.org/ seems promising, I'll try to find some cycles this
weekend to work on it.

**Comment by playground** [comment]
Thanks, please let me know when they are added.

**Comment by PeterKaminski09** [comment]
+1
